plugins/system/addmodulebutton/addmodulebutton.php

```diff
@@ -78,10 +78,11 @@ public function onBeforeRender()
 		if ($showAddModuleBtn)
 		{
 			// Add Script Options to pass the Button label Language Constant
-			$this->app->getDocument()->addScriptOptions('js-addModuleBtn', array(
-				'btnLabel' => Text::_('PLG_SYSTEM_ADD_MODULE_BUTTON_LABEL'),
-				'btnDescription' => Text::_('PLG_SYSTEM_ADD_MODULE_BUTTON_DESCRIPTION')
-			));
+			$config = array(
+				'btnLabel' 			=> Text::_('PLG_SYSTEM_ADD_MODULE_BUTTON_LABEL'),
+				'btnDescription' 	=> Text::_('PLG_SYSTEM_ADD_MODULE_BUTTON_DESCRIPTION')
```

Comment:
```suggestion
				'btnLabel'       => Text::_('PLG_SYSTEM_ADD_MODULE_BUTTON_LABEL'),
				'btnDescription' => Text::_('PLG_SYSTEM_ADD_MODULE_BUTTON_DESCRIPTION')
```
It seems you still haven't adjusted your editor or IDE to show you where you have tabs and where spaces. Here you have a mix of both. Please don't use tabs for such alignments inside of the code, e.g. for aligning the "=" of several assignment statements in consecutive lines. For such purpose use spaces. Tabs are used to indent code line from left, not to adjust code in the middle of a line somewhere.

Reply:
Oops I will configure my IDE as per your suggestion right away
Thanks